### ISSUE TITLE
Remove text -> rst building files

### ISSUE THREAD

**Issue by jaygambetta** [Qiskit/openqasm#60]
Since going forward all files should be rst i think we should remove all latex files. 

These include

.bib
.txt

**Comment by delapuente** [comment]
Agree, may the `.bib` file can stay since it is pretty standard for bibliography and it's supported by Sphinx. Perhaps the question is if we want to continue with this kind of references in the text. What do you think?
